ci/main.py

```diff
@@ -249,6 +252,27 @@ def scenario(self):
         else:
             Workflow.LOGGER.info("Skipped scenario's")
 
+    def configuration(self):
+        """
+        Execute custom configurations on a Open vStorage environment
+
+        Documentation:
+        https://github.com/openvstorage/framework/blob/5099afe52d67ae72d14286357b7706223c7bfd39/docs/scheduledtasks.md
+
+        :return: None
+        """
+        if 'configuration' in self.config['ci']:
+            if self.config['ci']['configuration']:
+                Workflow.LOGGER.info("Starting configuration")
+                if CelerySetup.override_scheduletasks(self.config['configuration']):
+                    Workflow.LOGGER.info("Finished configuration")
+                else:
+                    Workflow.LOGGER.warning("Failed to configure scheduled tasks")
+            else:
```

Comment:
if you merge the first 2 if's with an AND you can get rid of this extra else: ...